backend/tests/functional_tests/test_labels_repository.py

```diff
@@ -0,0 +1,111 @@
+"""Tests Labels Repository."""
+from typing import Any
+
+from medtagger.database.models import User
+from medtagger.definitions import LabelTool
+from medtagger.repositories import (
+    datasets as DatasetsRepository,
+    label_tags as LabelTagsRepository,
+    tasks as TasksRepository,
+    scans as ScansRepository,
+    labels as LabelsRepository,
+    users as UsersRepository,
+)
+from medtagger.types import LabelingTime
+
+
+def test_get_predefined_label_for_scan_in_task__no_predefined_label(prepare_environment: Any) -> None:
```

Comment:
I am wondering about double underscore. Is this a convention?

Reply:
This is not a convention but rather a "life-hack" that simplifies life 😉 For me, it is easier to read as I tested a single function in several different test cases.